src/services/storage/drivers/inMemory/index.ts

```diff
@@ -2,20 +2,16 @@ import cryptoRandomString from 'crypto-random-string'
 import { NotFoundError } from '../../../../errors/notFound.js'
 import { InMemoryStorageConfig } from '../../types/config.js'
 import type { StorageDriver } from '../../types/index.js'
-import type { StoredUrl } from '../../types/url.js'
+import type { StoredUrl, UrlWithInformation, UrlRequestData } from '../../types/url.js'
 
 export class InMemoryStorage implements StorageDriver {
-	// eslint-disable-next-line
-	constructor(public config: InMemoryStorageConfig) {}
-	data: { urls: Map<string, StoredUrl> } = {
+	data: { urls: Map<string, StoredUrl>; urlInformation: Map<string, UrlWithInformation> } = {
```

Comment:
I don't think we should save the entire URL inside the urlInformation.

Let's keep it the same as the Relational storage driver and place just the information there? In future versions, this should keep things simple when adding a feature across all drivers.

Reply:
Just get both, unite them and return it as UrlWithInformatiom.

Reply:
why to unite them? I can simply use only UrlWithInformation

Reply:
ok I'm changing it